test/fixtures/simple_example/main.tf

```diff
@@ -34,5 +34,16 @@ module "example" {
     "one" = true
     "two" = false
   }
+
+  lifecycle_rules = [{
```

Comment:
Remove, this is set in the example?

Reply:
@morgante  make sense to leave it in once place, you suggest to set it in example but not here, right?